Makefile

```diff
@@ -335,26 +335,68 @@ shadow-server: export TARGET := clojure
 shadow-server:##@ Start shadow-cljs in server mode for watching
 	yarn shadow-cljs server
 
+test: export TARGET := clojure
+test: export SHADOW_OUTPUT_TO := target/test/test.js
+test: export SHADOW_NS_REGEXP := .*-test$$
+test: ##@test Run all Clojure tests
+	yarn install
+	yarn shadow-cljs compile mocks && \
+	yarn shadow-cljs compile test && \
+	node --require ./test-resources/override.js target/test/test.js
+
 test-watch: export TARGET := clojure
-test-watch: ##@ Watch tests and re-run no changes to cljs files
+test-watch: export SHADOW_OUTPUT_TO := target/test/test.js
+test-watch: export SHADOW_NS_REGEXP := .*-test$$
+test-watch: ##@test Watch all Clojure tests and re-run when files change
 	yarn install
-	nodemon --exec 'yarn shadow-cljs compile mocks && yarn shadow-cljs compile test && node --require ./test-resources/override.js target/test/test.js' -e cljs
+	yarn shadow-cljs compile mocks && \
+	nodemon --exec 'yarn shadow-cljs compile test && node --require ./test-resources/override.js target/test/test.js' -e cljs
```

Comment:
Same, code duplication. Better to extract and parametrise.

Reply:
@yakimant, please, have a look at commit to give your second review pass https://github.com/status-im/status-mobile/pull/18304/commits/9c3459aaf1559e445a9b0f32ab0b81b61c0010ac. I could remove a substantial amount of code duplication between all clojure test targets. The small duplication between default target variables seems alright to me, but is that good enough for you?

Reply:
Thanks @ilmotta, I would move `export TARGET := clojure` it possible too.
If not - I'm ok to merge.